### PR TITLE
[Edge ] SDO interlocking

### PR DESCRIPTION
**Pull request by ASawwaf**
SDO interlocking

### CODE REVIEW

File: doc/tech/usecases.md [hunk 1]
```diff
@@ -208,10 +208,10 @@ CNTT believes that Edge computing is unique in terms of infrastructure requireme
 <a name="5.1.7"></a>
 ### 5.1.7 SDO interlocking
 -	OpenStack Edge Computing Group 
-- Linux Foundation - Edge ( LF-Edge)
-- GMSA - Opertaor Platform Group & Telco Edge Cloud
+- Linux Foundation - Edge (LF-Edge)
+- GMSA - Opertaor Platform Group (OPG)& Telco Edge Cloud (TEC)
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:212]: Opertaor -> Operator

**Comment by rabi-abdel** [comment]
@CsatariGergely can you please re-review?

**Comment by ASawwaf** [comment]
@walterkozlowski  thx i fixed it , for SDO  this for the current Edge engagement bodies,  ONF we are working with them in CNTT in general and Networking part in specific

**Comment by rabi-abdel** [comment]
@walterkozlowski can you please re-review?

**Pull request merged**